conbench/api/results.py

```diff
@@ -128,34 +134,44 @@ def get(self):
         tags:
           - Benchmarks
         """
+        # Note(JP): "case name" is the conceptual benchmark name. Interesting,
+        # so this is like asking "give me results for this benchmark".
         if name_arg := f.request.args.get("name"):
+            # TODO: This needs a limit, and sorting behavior.
+            # arbitrary limit for now.
             benchmark_results = BenchmarkResult.search(
                 filters=[Case.name == name_arg],
                 joins=[Case],
             )
-            # Since there's no limit on the number of BenchmarkResults, we could take
-            # forever calculating z-scores with no caching advantage. So don't do that.
-            for benchmark_result in benchmark_results:
-                benchmark_result.z_score = None
```

Comment:
Instead of this, I found it a little cleaner to (for now) handle AttributeError in the to-dict function.